src/components/time-entries/TimeEntries.tsx

```diff
@@ -43,48 +54,59 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
     <>
       <Styled.Main>
         <Styled.Container>
+          <Styled.Label>Filter by client: </Styled.Label>
+          <Styled.Select value={selectedClient} onChange={filterClients}>
+            <option disabled value="">
```

Comment:
Here you can remove `disabled` ;) Because you don't want the filter to always stay active once you've chosen a value.